examples/cd_mol.ipynb

```diff
@@ -0,0 +1,1071 @@
+      "Drift? No!\n",
+      "p-value: [0.13626936]\n",
+      "\n",
+      "imbalanced sample ROC-AUC: 0.749 -- loss: 0.940\n",
+      "Drift? Yes!\n",
+      "p-value: [4.5092685e-05]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "xs = {'H0': x_h0, 'test sample': sample('test', 500), 'imbalanced sample': x_imb}\n",
+    "make_predictions(dd, xs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, prediction distribution shift is detected for the imbalanced sample but not for the random test sample with similar label distribution as the reference data.\n",
+    "\n",
+    "### Prediction uncertainty drift\n",
+    "\n",
+    "The [model uncertainty drift detector](https://docs.seldon.io/projects/alibi-detect/en/latest/methods/modeluncdrift.html) can pick up when the model predictions drift into areas of changed uncertainty compared to the reference data. This can be a good proxy for drift which results in model performance degradation. The uncertainty is estimated via a Monte Carlo estimate ([MC dropout](https://arxiv.org/abs/1506.02142)). We use the [RegressorUncertaintyDrift detector](https://docs.seldon.io/projects/alibi-detect/en/latest/methods/modeluncdrift.html) since our binary classification model returns 1D logits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:alibi_detect.cd.utils:Input shape could not be inferred. If alibi_detect.models.tensorflow.embedding.TransformerEmbedding is used as preprocessing step, a saved detector cannot be reinitialized.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from alibi_detect.cd import RegressorUncertaintyDrift\n",
+    "\n",
+    "dd = RegressorUncertaintyDrift(x_ref, model=model, backend='pytorch', p_val=.05, n_evals=100,\n",
+    "                               uncertainty_type='mc_dropout', preprocess_batch_fn=batch_fn)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "H0 ROC-AUC: 0.763 -- loss: 0.208\n",
+      "Drift? No!\n",
+      "p-value: [0.52494323]\n",
+      "\n",
+      "test sample ROC-AUC: 0.736 -- loss: 0.150\n",
+      "Drift? Yes!\n",
+      "p-value: [0.]\n",
+      "\n",
+      "imbalanced sample ROC-AUC: 0.749 -- loss: 0.940\n",
+      "Drift? Yes!\n",
+      "p-value: [0.]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "make_predictions(dd, xs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Although we didn't pick up drift in the GIN model prediction distribution for the test sample, we can see that the model is less certain about the predictions on the test set, illustrated by the lower ROC-AUC.\n",
```

Comment:
It looks to me like drift is picked up for the test sample?

Reply:
No, this refers to the prediction distribution drift in the previous section, not the prediction uncertainty drift in this section. But it is clearly unclear so I'll clarify the sentence. :)